challenge-10/challenge-10.js

```diff
@@ -50,8 +69,19 @@ parâmetros;
 operador passado para a função "calculator", e passando para esse método
 os dois parâmetros da função de retorno de "calculator".
 */
-// ?
-
+	function calculator(operador) {
+		if( isOperatorValid(operador) === false ) {
+			return false;
+		} {
```

Comment:
então mas calma ai, ainda to com a duvida. hahaha
vamos supor que eu tenho um estrtura if que eu preciso de uma segunda validação ou seja de um else:
if(expressão) {
 return blablalba
} (não existe a necessidade da plavra else) { 
 return bla bla bla
} 

sem return: 
if(expressão) {
 console.log(bla bla bla)
} else (por não ter return existe a necessidade da plavra else) { 
 return bla bla bla
};

eu sei que nesse execicio não precisa do else  mas minha a duvida é quanto a linguagem, quando houver necessidade de uma outra validação